include/CoreGraphics/CGColor.h

```diff
@@ -40,3 +40,5 @@ COREGRAPHICS_EXPORT size_t CGColorGetNumberOfComponents(CGColorRef color);
 COREGRAPHICS_EXPORT const CGFloat* CGColorGetComponents(CGColorRef color);
 COREGRAPHICS_EXPORT CGPatternRef CGColorGetPattern(CGColorRef color);
 COREGRAPHICS_EXPORT CFTypeID CGColorGetTypeID();
+COREGRAPHICS_EXPORT CGColorRef CGColorCreateGenericRGB(CGFloat red, CGFloat green, CGFloat blue, CGFloat alpha);
```

Comment:
That would be done by an extra issue.

---
In reply to: [112492607](https://github.com/Microsoft/WinObjC/pull/2538#discussion_r112492607) [](ancestors = 112492607)